applications/asset_tracker_v2/src/cloud/nrf_cloud_integration.c

```diff
@@ -155,7 +155,7 @@ static int send_service_info(void)
 }
 
 /* Function used to filter out responses to cellular position requests. */
-static int cell_pos_response_filter(char *response, size_t len)
+int cell_pos_response_filter(char *response, size_t len)
```

Comment:
@simensrostad
Some of these cloud integration files don't have license information. Is this intentional and how can we get around the license check?

PS. I cannot comment this on the first line of this file as there are no changes so adding it here.

Reply:
No, its not intentional. Rather a mishap. Just add them 😄

Reply:
I'll add them if they are not there when I'm closer to trying to get something merged. However, this is getting a big PR for ATv2 even if I end up doing additional PRs for location library etc. Hence, if your team has PRs open for ATv2, you could check the license issues in cloud integration files and add the licenses as part of some smaller PR.

Reply:
Seems license is now in place in this file. However, there are aws and azure integration files that doesn't have license. I won't add them in this PR as I'm not modifying those files. there are certainly smaller PR's for ATv2 that can add them.